examples/tomo/checks/check_axes_cone2d_bp.py

```diff
@@ -0,0 +1,64 @@
+"""Cone beam 2D example for checking that orientations are handled correctly.
```

Comment:
I added a README for this folder, implemented the phantom for 2d and removed the workaround, renamed the angle things and added global variables for `impl` and `shift` for the user.

What I find hard is to reduce the complexity. These axis convention things just are complex by nature, and the comments try to explain what happens in a given setting. I added sections for Spyder so that users can execute the code section by section, which makes more sense (one comparison at a time).